api/server.spec.js

```diff
@@ -25,17 +26,128 @@ describe('The API Server', () => {
     //   // write a beforeEach hook that will populate your test DB with data
     //   // each time this hook runs, you should save a document to your db
     //   // by saving the document you'll be able to use it in each of your `it` blocks
+    return Game.create({
+        title: 'California Games',
+        genre: 'Sports',
+        releaseDate: 'June 1987'
+      })
+      .then(game => {
+        gameId = game._id
```

Comment:
Very nice! I like that you're saving the `_id` here.

We could also use `async`/`await` here since we're already doing that in the rest of this file:

```js
beforeEach(async () => {
  const game = await Game.create({
      title: 'California Games',
      genre: 'Sports',
      releaseDate: 'June 1987'
    })
  gameId = game._id
})
```